components/crud-web-apps/tensorboards/Dockerfile

```diff
@@ -48,4 +48,4 @@ COPY ./tensorboards/backend/Makefile .
 
 COPY --from=frontend /src/dist/ /src/app/static/
 
-ENTRYPOINT make run
+ENTRYPOINT gunicorn -w 3 --bind 0.0.0.0:5000 --access-logfile - entrypoint:app

```

Comment:
Will this have any incompatibilities for IPv6?

Reply:
@nrchakradhar Good point. These same entrypoint is being used by the Jupyter-Web-App. Do you have suggestions how to make this IPv6 compatible?

Reply:
I do not know anything about gunicorn, but from its documentation, the following may work:
https://docs.gunicorn.org/en/stable/settings.html
Multiple addresses can be bound. ex.:

$ gunicorn -b 127.0.0.1:8000 -b [::1]:8000 test:app
will bind the test:app application on localhost both on ipv6 and ipv4 interfaces.

I found an interesting observation here(its quite old though)
https://github.com/benoitc/gunicorn/issues/1138#issuecomment-158358666

Can we make this configurable from manifests so that IPv6 support is provided. There are few issues created already for IPv6 support: #5615 #5605

Reply:
@nrchakradhar thank you for your comments. I'm planning on tackling the IPv6 support for all the web apps after 1.3.

The first step is to bind the IPv6 equivalent addresses as well, as you've pointed out in your links. But I want to do a good deep dive on this and test some edge cases for which I don't have the cycles right now.